style.css

```diff
@@ -8,6 +8,7 @@ body {
   background-color: white;
   margin: 0;
   padding: 0;
+  width: 100%;
 }
 
 .container {
```

Comment:
- [ ] As mentioned by the previous reviewer and for the project requirements, kindly make sure your application is responsive and the design sticks as much as possible to the one from your chosen design. Below you can see hoew your application behaviors when changing the screen size.
![git](https://user-images.githubusercontent.com/77843050/244030408-b193363a-9aee-4328-bace-d0136ca33218.gif)